docs/TranslationLocales.md

```diff
@@ -15,6 +15,7 @@ You can find translation packages for the following languages:
 - Catalan (`ca`): [joshf/ra-language-catalan](https://github.com/joshf/ra-language-catalan)
 - Chinese (`zh-TW`): [areyliu6/ra-language-chinese-traditional](https://github.com/areyliu6/ra-language-chinese-traditional)
 - Chinese (`zh`): [chen4w/ra-language-chinese](https://github.com/chen4w/ra-language-chinese)
+- Chinese (`zh`): [haxqer/ra-language-chinese](https://github.com/haxqer/ra-language-chinese)
```

Comment:
What is the difference with the existing package?

Reply:
`chen4w/ra-language-chinese` unable to support the latest ReactAdmin version.
The new version of RA has added some fields.

`haxqer/ra-language-chinese` support the latest ReactAdmin version.

Reply:
I see, can you please remove the old one then?

Reply:
Done